### ISSUE TITLE
[AIR 33.1.1.674][Windows] AIR applications, ADL and ADT cannot be started if Scout launched

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#1355]
## Problem Description
AIR 33.1.1.674 applications, `adl` and `adt` cannot be started if Adobe Scout launched.
Tested with Windows 10 devices. Didn't test it with other platforms yet.
It works fine with AIR 33.1.1.633 and below.

Any AIR application built with AIR SDK 33.1.1.674 cannot started if Scout lauched. Also you cannot test AIR applications with IDE if Scout lauched. Also you cannot even package AIR applications command line if Scout lauched.

## Steps to Reproduce
Launch Adobe Scout and try to package any AIR application with AIR SDK 33.1.1.674 via command line or try to test any AIR application using VScode or Adobe Animate.

**Actual Result:**
With command line you will get error:
```
unexpected failure: Platform conversion failed. Process exited with error -1073740791. Output was ""
java.io.IOException: Platform conversion failed. Process exited with error -1073740791. Output was ""
        at com.adobe.air.nai.NativePackager.conversionFail(NativePackager.java)
        at com.adobe.air.nai.NativePackager.invokePlatformConversion(NativePackager.java)
        at com.adobe.air.nai.NativePackager.createPackage(NativePackager.java)
        at com.adobe.air.ADT.parseArgsAndGo(ADT.java)
        at com.adobe.air.ADT.run(ADT.java)
        at com.adobe.air.ADT.main(ADT.java)
```
Application will not be packaged or run. You will get error: `Process exited with error -1073740791`

**Expected Result:**
Application will be packaged or run without errors.

## Known Workarounds
none
*Use AIR 33.1.1.633 or below.

**Comment by ajwfrost** [comment]
Thanks for the report ... so, we had an issue found during the QA/testing for the release, on macOS and Android. The problem didn't show up on Windows though so we didn't update those binaries -> but it appears that having a connection to Scout is then triggering this problem even on Windows.

We just confirmed that a build off the latest code has fixed the problem so will rebuild the Windows DLLs and update them in the package, and re-upload to the website...

**Comment by ajwfrost** [comment]
FYI: should be sorted out now with the updated release on the website...
thanks

**Comment by itlancer** [comment]
@ajwfrost
Thanks for quick fix! Now it works fine..

**Comment by itlancer** [comment]
@ajwfrost, the same problem for iOS even with updated SDK.

**Comment by itlancer** [comment]
Finally fixed with latest AIR SDK 33.1.1.686 for iOS too.
Thanks!
